src/parsers/power_system_table_data.jl

```diff
@@ -1020,6 +1020,7 @@ function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames,
         time_limits = timelimits,
         operation_cost = op_cost,
         base_power = base_power,
+        must_run = gen.must_run,
```

Comment:
where is the must run info coming from?

Reply:
It is defined in power_system_structs.json and the user_descriptors.yml of the data, so the data would come from gen.csv. I'm not sure if that is the answer you're looking for.

Reply:
But that value isn't been read and passed to the `ThermalStandard` call.

Reply:
Yeah, gen has a field must_run because it is defined in both user_descriptors and power_system_inputs.

Reply:
Yes, that value is read from the CSV into the gen dict but you aren't reading it from that dict. Did you test this?

Reply:
I'm running some more tests now with Pedro. I made this a draft for now. I will open a PR once I get the go ahead from Pedro.

Reply:
I tested this on Pedro's ERCOT test data. This does what it is supposed to in general but will fail due to CSV data parsing. For instance, I printed gen.must_run for his data for debugging purposes and sometimes Bool is read as an InlineString.
gen_must_run = false
gen_must_run = InlineStrings.String7("False")

Reply:
Why not fix it in this PR?